client/my-sites/plugins/plugin-install-button/index.jsx

```diff
@@ -147,6 +148,24 @@ export class PluginInstallButton extends Component {
 		return null;
 	}
 
+	renderMarketplaceButton() {
+		const { translate, selectedSite, plugin, billingPeriod } = this.props;
+		const variationPeriod = getPeriodVariationValue( billingPeriod );
+		const product_slug = plugin?.variations?.[ variationPeriod ]?.product_slug;
+
+		return (
+			<span className="plugin-install-button__install embed">
+				<Button
+					href={ `/checkout/${ selectedSite.slug }/${ product_slug }?redirect_to=/marketplace/thank-you/${ plugin.slug }/${ selectedSite.slug }#step2` }
+				>
+					<Gridicon key="plus-icon" icon="plus-small" size={ 18 } />
+					<Gridicon icon="plugins" size={ 18 } />
+					{ translate( 'Purchase and activate' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **42** times:
`translate( 'Purchase and Activate &raquo;' )` *ES Score: 7*